managed/src/main/java/com/yugabyte/yw/models/YugawareProperty.java

```diff
@@ -44,9 +46,10 @@
   // The property config.
   @Constraints.Required
   @Column(columnDefinition = "TEXT")
-  private JsonNode value;
+  @DbJson
+  private Map<String, Object> value;
```

Comment:
This is a true Json Node. So no need to fix this.

Reply:
how to identify this?

Reply:
Follow the trail for where it is getting set.  it is a json comming from some external tool. For example health check script or in this case some python script that is storing too much variety.
Of course you can follow the trail into that script and deduce that it has some fixed format that can be captured by a POJO. in that case we can use that POJO. 
Or you see that it is being updated as a map by that tool.

Map<String, String> is better than ObjectNode as value wont be another objectNode or arrayNode.
But Map<String, Object> does not buy us any additional type safety or contract beyond ObjectNode. 
It is just an equivalent representation that is equially opaque.

But in API having Json node is useless and Map<String Object> will be better. To achieve that we can simply use 
`@ApiModelProperty(dataType="Map")`
or 
`@ApiModelProperty(dataType="Object")`

And we will get to enter a JsonObject equivalent.